old/articles/toc.yml

```diff
@@ -0,0 +1,81 @@
+- name: Overview
```

Comment:
Maybe we just delete this whole file, it will still exist in the Git history.

Reply:
Agreed, we can remove the OLD folder

Reply:
Cool, should I do the commit to remove it?